templates/telegraf.conf.erb

```diff
@@ -24,16 +24,12 @@
   debug = <%= @debug %>
   quiet = <%= @quiet %>
 
-<% if @_outputs -%>
```

Comment:
A hash is always `truthy`.  I did consider changing it to `unless @outputs.empty?`, but I don't think there's any point.  The defaults aren't empty, and does having no inputs or outputs make a lot of sense?